test/fixtures/todo/json5/serialization/object.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'object with numeric literals',
```

Comment:
@lundibundi, saying "numeric keys" is incorrect, it should be "numeric literals as keys".
Also, you've left this comment in the wrong PR, this change is from https://github.com/metarhia/jstp/pull/237.